design/one-pager-change-logs.md

```diff
@@ -0,0 +1,432 @@
+this difference.
+
+This means that it is possible for Crossplane to make changes to external
+systems even when no user is actively interacting with the control plane.
+Consider the scenario when someone updates a resource outside of Crossplane,
+e.g. via the AWS console or `gcloud` CLI. When Crossplane detects this
+"configuration drift", it will enforce its "source of truth" to eventually
+correct this unexpected change without any user interaction with the control
+plane.  Currently, there is no mechanism for users of Crossplane to have
+visibility into these types of "unexpected" changes that Crossplane is reverting
+on their behalf.
+
+With Crossplane acting continuously and autonomously to update critical
+infrastructure, it is vital for users to have insight into the operations being
+performed, so they can build and maintain a strong sense of confidence and trust
+in their control planes.
+
+There are many approaches that can (and should) be taken to build trust along
+the various phases of the lifecycle of control planes and the infrastructure
+that they manage, but this design will focus specifically on the observability
+of changes that the control plane makes to external systems at runtime. This is
+not intended to facilitate review or approve the changes before they are made,
+but rather to provide an auditable record of all changes that have been
+performed.
+
+This means that Crossplane (and its Providers) should start to expose data about
+all changes that the control plane is making to external systems, with as much
+detail as possible so the platform team can understand what is changing, for
+what reason it is changing, and the outcome of the change. With this
+information, Crossplane users will be able to build deeper trust and confidence
+in the operation of their control planes, as well as have forensic data
+available to them in the event that an unexpected change occurs.
+
+## Proposal
+
+### Change Log Entry Format
+
+The change log will be a series of individual entries that capture state about
+the resources involved in every operation a provider performs on its
+external system. Later in this document we will explore HOW the logs are
+generated and stored, but for now we will focus on WHAT will be collected.
+
+As a general principal, we will be collecting raw observation data that has not
+been processed in any significant way. A system of the users choosing that
+collects the logs entries can perform opinionated calculations, analysis, or
+alerting that is specific to their operating environment.
+
+Each entry in the change log will contain the following data:
+
+| Field Description | Example |
+|-------------------| ------- |
+| Timestamp | ISO 8601 format, e.g. `2023-04-01T12:34:56Z` |
+| Provider name | `xpkg.upbound.io/upbound/provider-aws-ec2:v1.8.0` |
+| Type (GVK) | `apiVersion: ec2.aws.upbound.io/v1beta2, kind: Instance` |
+| Name | `dev-instance-bt66d` |
+| External Name | `vpc-4fa03d9fb92dfec50` |
```

Comment:
Any chance we can include resource labels?

Assume we aggregate these changelog events into a search index.  We'd then want to correlate them with higher-level events (ArgoCD or FluxCD resource events) and finally point to a commit (or set of commits).

Having the resource labels as KVs in this schema would allow us to build the relationship with those higher-level events more directly, providing a solid end-to-end view into how changes at the higher-end of the abstraction flow to changes at the lowest end.